### DISCUSSION TITLE
Twitter bot

### DISCUSSION THREAD

**Discussion by kleinpetr** [mastodon/mastodon#18367] [Q&A] [answered]
I know that there is an option to create Mastodon bot from Twitter account, but is there any vice-versa possibility? Means create Twitter bot which replicates Mastodon toots :)

**Comment by dsmouse** [comment, marked as answer]
I know external services, such as [@moaparty@fosstodon.org](https://fosstodon.org/@moaparty) can do both directions and can do conditional based on tags

**Comment by kleinpetr** [comment]
Awesome! thank you so much :)
